dimod/discrete/discrete_quadratic_model.py

```diff
@@ -663,14 +663,17 @@ def _to_file_numpy(self, file, compressed):
         file.write(np.dtype('<u4').type(end - start).tobytes())
         file.seek(end)
 
-    def to_file(self, compressed=False, ignore_labels=False,
+    def to_file(self, compress=False, compressed=None, ignore_labels=False,
```

Comment:
To preserve function signature for positional args, it's better to just append the new argument. In theory, this change breaks the public API, and would require a major version increment :grimacing:

Reply:
Sorry, I figured it's unlikely anyone was already using this, so I preferred the least confusing signature and to ensure a deprecation notice if there is someone using it already.

Reply:
No, absolutely, this was just an FYI comment. I'm pretty sure this change won't break anyone's code.